### PR TITLE
Jv lottery HW solution

### PR DESCRIPTION
**Pull request by luk14n**

### CODE REVIEW

File: src/main/java/core/basesyntax/ColorSupplier.java [hunk 1]
```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
-public class ColorSupplier {
+import java.util.Random;
+
+public class ColorSupplier extends Ball {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Colors.values().length);
```
Comment (Rommelua) [inline src/main/java/core/basesyntax/ColorSupplier.java:7]: image

File: src/main/java/core/basesyntax/Colors.java [hunk 2]
```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public enum Colors {
```
Comment (Rommelua) [inline src/main/java/core/basesyntax/Colors.java:3]:
```suggestion
public enum Color {
```

File: src/main/java/core/basesyntax/Lottery.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    private static final Random random = new Random();
+
+    public Ball getRandomBall() {
+        String randomColor = getRandomColor();
+        int randomNumber = random.nextInt(100);
```
Comment (Rommelua) [inline src/main/java/core/basesyntax/Lottery.java:10]: image

File: src/main/java/core/basesyntax/Lottery.java [hunk 4]
```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
```
Comment (Rommelua) [inline src/main/java/core/basesyntax/Lottery.java:5]:
```suggestion
public class Lottery {
```

**Review by Rommelua — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/Lottery.java (hunk 4 above)
Comment (luk14n) [reply src/main/java/core/basesyntax/Lottery.java:5]: oh, I see. But when I'm trying to run it without 'extends ColorSupplier' I have compiler error. Here are the screenshot
Extends_Issue

**Review by sokimaaa — APPROVED** [review]

**Pull request closed without merging**